Alertmanager configuration process updates (#78)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_configuring-custom-alerts.adoc

[id="configuring-custom-alerts_{context}"]
= Configuring custom alerts

> Custom alerts can be added to the `PrometheusRule` object created in <<creating-an-alert-rule-in-prometheus_advanced-features>>:

Reviewer: ```suggestion
You can add custom alerts to the `PrometheusRule` object that you created in <<creating-an-alert-rule-in-prometheus_advanced-features>>:
```

Author: Huh I was specifically avoiding that narrator position. Maybe I've misunderstood what voice I'm aiming for this whole time?

Reviewer: It's very easy to fall into passive voice but active voice is always the preferred voice. It reduces confusion and is more prescriptive
